internal/pkg/cli/app_package.go

```diff
@@ -112,25 +126,40 @@ func (o *packageAppOpts) Ask() error {
 
 // Execute prints the CloudFormation template of the application for the environment.
 func (o *packageAppOpts) Execute() error {
+	if o.OutputDir != "" {
+		if err := o.setAppFileWriters(); err != nil {
+			return err
+		}
+	}
+
 	env, err := o.store.GetEnvironment(o.ProjectName(), o.EnvName)
 	if err != nil {
 		return err
 	}
 
-	if o.OutputDir != "" {
-		if err := o.setFileWriters(); err != nil {
-			return err
-		}
+	appTemplates, err := o.getAppTemplates(env)
+	if err != nil {
+		return err
+	}
+	if _, err = o.stackWriter.Write([]byte(appTemplates.stack)); err != nil {
+		return err
+	}
+	if _, err = o.paramsWriter.Write([]byte(appTemplates.configuration)); err != nil {
+		return err
 	}
 
-	templates, err := o.getTemplates(env)
+	addonsTemplate, err := o.getAddonsTemplate()
+	// return nil if addons dir doesn't exist.
+	if _, ok := err.(*workspace.ErrAddonsDirNotExist); ok {
+		return nil
+	}
```

Comment:
The safer way to error check is I believe with the newer `errors.Is` and `errors.As` functions:
```go
var notExistErr *workspace.ErrAddonsDirNotExist
if errors.As(err, &notExistErr) {
      return nil
}
```
https://blog.golang.org/go1.13-errors